### PR TITLE
all: Update code of `platform` and `anchor-reference-server` so the SEP-38 implementation becomes compliant with the validation tool

### LINKED ISSUE #116: Validate Sep38 with anchor validation tool
Run platform server and reference server and verify SEP38 with https://anchor-tests.stellar.org/
GanttStart: 2022-03-29
GanttDue: 2022-04-01

### PR DESCRIPTION
**Pull request by marcelosalloum**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Update code of `platform` and `anchor-reference-server` so the SEP-38 implementation becomes compliant with the validation tool https://github.com/stellar/stellar-anchor-tests.

### Why

As part of #116.

### Further discussion

I'll add a GH Action to execute the validation tool https://github.com/stellar/stellar-anchor-tests against this project in the next PR.

### CODE REVIEW

**Review by lijamie98 — APPROVED** [review]
LGTM

File: platform/src/main/resources/default-config.yaml [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
     enabled: true
     homeDomain: localhost:8080
     clientAttributionRequired: false
-    clientAttributionAllowList: lobstr.co,preview.lobstr.co
+    clientAttributionAllowList: lobstr.co,preview.lobstr.co # use this if we want to white list
+    # clientAttributionDenyList: # use this if we want to black list.
     authTimeout: 900
     jwtTimeout: 86400
-    signingSeed:
+    signingSeed: SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
```
Comment (JakeUrban) [inline platform/src/main/resources/default-config.yaml:50]: We shouldn't provide a secret key here should we?

**Review by JakeUrban — APPROVED** [review]
LGTM other than the secret key in the config.

File: platform/src/main/resources/default-config.yaml (hunk 1 above)
Comment (marcelosalloum) [reply platform/src/main/resources/default-config.yaml:50]: This is already being used in other yaml files. We still have to figure out how to inject this using Kubernetes secrets.

Comment (marcelosalloum) [reply platform/src/main/resources/default-config.yaml:50]: We have this TODO in kube the k8s repo: https://github.com/stellar/kube/blob/fec12a77e08c911d9799bc042414b891ea1076a3/kube001-dev/namespaces/common-previews/anchor-platform-dev/sep-server-config-map.yaml#L9

**Pull request merged**